pages/linux/llvd.md

```diff
@@ -0,0 +1,20 @@
+# llvd
+
+> Linkedin Learning Video Downloader.
+> More information: <https://github.com/knowbee/llvd>.
+
+- Download a [c]ourse using cookie-based authentication:
+
+`llvd -c {{course-slug}} --cookies`
+
+- Download a course at a specific [r]esolution:
+
+`llvd -c {{course-slug}} -r 720`
+
+- Download a course with [ca]ptions (subtitles):
```

Comment:
Just a small suggestion
```suggestion
- Download a course with captions (subtitles):
```
Here, captions isn't a valid flag as `--ca` doesn't work.

> Yeah. But, is there a separate syntax to write suggestions?

You could refer to our [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md) for information on writing command descriptions, syntax and other information.

Reply:
> Here, captions isn't a valid flag as `--ca` doesn't work.

Try `-ca`

Reply:
Yep, ca works. I will resolve this conversation then.
Ref. 
![image](https://user-images.githubusercontent.com/26346867/192001560-f799633f-3383-44de-9395-cf7b7a84d424.png)